software/index.html

```diff
@@ -36,96 +37,133 @@
-                        <p class="fs-5 text-start">
-                            The program consists of 7 weekly 2 hour learning sessions on Saturday (hence our name!) with weekly review and help hours. All this leads in our final project, a fully-featured Pokedex app using public APIs.
-                            This project is great on a resume or portfolio and will spark interest from recruiters and hiring managers!
-                        </p>
-                    </div>
-                    <div class="col-md-2 align-self-center">
-                        <img class="img-fluid" src="images/react_logo.svg" alt="ReactJS Logo">
                     </div>
                 </div>
-                <p class="fs-5 text-center">Contact us at <a class="link-dark fw-bold" href="mailto:software-saturdays@purdueieee.org">software-saturdays@purdueieee.org</a> with questions.</p>
-                <div class="d-grid gap-2 d-sm-flex justify-content-sm-center text-center">
-                    <a href="/software/attendee/" type="button" class="btn btn-ieee btn-lg px-4 gap-3">Register Today!</a>
-                </div>
             </div>
 
-            <div class="my-2 p-5 container-fluid bg-purdue">
-                <div class="container-lg">
-                    <h3 class="text-center">Frequently Asked Questions:</h3>
-                    <ul class="fs-4">
-                        <li class="my-3"><span class="fw-bold">I am not in the College of Engineering, can I still join?</span><br>Of course, any Purdue student can join, regardless of major, college, or class standing!</li>
-                        <li class="my-3"><span class="fw-bold">How much time does this program require?</span><br>Aside from the two hour weekly learning session, most people spend only 1-2 hours reviewing material.</li>
-                        <li class="my-3"><span class="fw-bold">How much prior experience do I need?</span><br>Absolutely none. We teach you what you have to know! If you are familiar with web development, we do offer an intermediate track that may be more your style.</li>
-                        <li class="my-3"><span class="fw-bold">What is this certificate you mentioned?</span><br>Any attendee who completes our final project will recieve a digital certificate from the College of Engineering,
-                            with a QR code to verify your achievement.</li>
-                    </ul>
-                </div>
+             <!-- Mentors -->
+            <div class="mt-2 mb-5 container-fluid text-light text-center fade-in-top">
+                <h1 class="text-center mb-3"><strong>Mentors</strong></h3>
+                <div class="container-fluid bg-secondary extra-rounded mb-5">
+                    <div class="d-flex flex-column pt-5">
+                        <div class="d-flex flex-row justify-content-center">
+                            <img src="images/chair_profile.png" class="circle-img-frame mb-4" width="30%" alt="ReactJS Logo">
```

Comment:
Picture is too large, also using wrong photo - one is already included in the "Officers" page.